### PR TITLE
Various minor fixes and additional details

### PR DESCRIPTION
**Pull request by ghost**
processes/development/new-app-setup/README.md

I've tried to clarify a bunch of things that seemed vague or ambiguous to me. Added some additional details. Etc.

### CODE REVIEW

**Comment by ghost** [comment]
Please wait to review this. I have more changes coming.

**Review by pauldowman — CHANGES_REQUESTED** [review]
All of the lists with indented sub-lists are broken, they render as if they're all one top-level list. I think Markdown sub-lists are supposed to be 3 spaces.

Click the "view button" on the README.md file in this PR to see what I mean.

**Comment by ghost** [comment]
Fixed.

File: processes/development/new-app-setup/README.md [hunk 1]
```diff
@@ -1,46 +1,84 @@
...
-1. Create production Heroku app (same steps as staging)
+        1. Disable email for all notifications and add the Slack web hook URL
+            * **NOTE** This is a process that involves editing each notification. (TODO: details)
     1. Configure Mailgun add-on
-      2. Add DNS records
-      3. Edit `MAILGUN_*` env vars to contain the info for the verified domain (instead of the sandbox domain).
-    1. If you are configuring dynos Heroku requires a `Procfile` where you specify the startup process commands. Here is a basic meteor example [Procfile](procfile-example)
+        1. Add DNS records (TODO: more details)
+        1. Edit `MAILGUN_*` env vars to contain the info for the verified domain (instead of the sandbox domain).
+    1. ???? If you are configuring dynos Heroku requires a `Procfile` where you specify the startup process commands. Here is a basic meteor example [Procfile](procfile-example)
```
Comment (pauldowman) [inline processes/development/new-app-setup/README.md:54]: You don't need this. The Meteor buildpack takes care of it.

Comment (pauldowman) [reply processes/development/new-app-setup/README.md:54]: @ccuilla I think we should remove this line.

**Pull request merged**